apps/studio/src/server/trpc.ts

```diff
@@ -11,9 +11,9 @@
 import superjson from 'superjson'
 import { ZodError } from 'zod'
 import { TRPCError, initTRPC } from '@trpc/server'
+import { type OpenApiMeta } from 'trpc-openapi'
```

Comment:
understand this is related to the vapt finding of the content header, could i trouble you to add PR description for context for engs?

Reply:
Updated!